src/doc/src/reference/profiles.md

```diff
@@ -457,6 +460,26 @@ match wins):
 
 Overrides cannot specify the `panic`, `lto`, or `rpath` settings.
 
+Overrides inherit to custom profiles and take precedence unless explicitly overriden.
+For example, in the below example package `foo` will still be built with `opt-level = 1`
```

Comment:
This doesn't seem to match the current behavior. Check this test as a reference:

https://github.com/rust-lang/cargo/blob/61855e72a2200ca3d39785c80f57719f856a40ce/tests/testsuite/profile_custom.rs#L295-L364